server/src/test/java/org/elasticsearch/common/collect/ImmutableOpenMapTests.java

```diff
@@ -120,6 +131,58 @@ public void testEmptyKeySetWorks() {
         assertThat(ImmutableOpenMap.of().keySet().size(), equalTo(0));
     }
 
+    public void testEmptyValuesIsCollection() {
+        assertTrue(ImmutableOpenMap.of().values().isEmpty());
+    }
+
+    public void testValuesIsCollection() {
+        assertTrue(countryPopulations.values().containsAll(List.of(37_846_611, 46_754_778, 60_461_826, 65_273_511, 83_783_942)));
```

Comment:
Hamcrest has helpers to check collections containing values, please use them, they give much better failure messages